src/plugins/gui/creditCardPlugin.js

```diff
@@ -170,9 +172,11 @@ const ERROR_TEXT = {
   assetUnsupported: s.strings.fiat_plugin_asset_unsupported
 }
 
-export const getBestError = (errorQuotes: FiatProviderQuoteError[], currencyCode: string): string | void => {
+export const getBestError = (errorQuotes: FiatProviderError[], currencyCode: string): string | void => {
```

Comment:
The `errorQuotes` array will eventually need to be `mixed[]`, since we don't actually know what is being thrown. I think TypeScript will eventually complain if / when we upgrade to that, but for now Flow is happy so we can leave it. The overall architecture is correct, even if the types are loose.